tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -104,6 +104,11 @@ public Row(String name, int partition, Long timestamp) {
             this.timestamp = timestamp;
         }
 
+        @Override
+        public String toString() {
+            return "Row[name:" + name + ",partition:" + partition + ",timestamp:" + timestamp;
```

Comment:
Is it used for debugging?

Reply:
Could you rename the `timestamp` to `offset`?

Reply:
you mean like `return "Row[name:" + name + ",partition:" + partition + ",offset:" + offset;` ?

Reply:
yep, the output from shell is offset rather than timestamp

Reply:
I tried to do the refactoring to use `offset` rather than `timestamp`, but I'm a bit lost as in how I can get the offset number from the `GetOffsetShell`. I tried to read through the CI output but couldn't locate  where the output is an offset rather than the timestamp. Could you point me to the outputs that you saw so that I can get more context of it ? Thanks

Reply:
https://github.com/apache/kafka/blob/trunk/tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java#L93

Reply:
Thanks. Amended to use offset instead of timestamp